src/IR.h

```diff
@@ -509,11 +509,14 @@ struct Call : public ExprNode<Call> {
         div_round_to_zero,
         dynamic_shuffle,
         extract_mask_element,
-        gather,
         glsl_texture_load,
         glsl_texture_store,
         glsl_varying,
         gpu_thread_barrier,
+        hvx_gather,
```

Comment:
Maybe this shouldn't be in the intrinsics list at all? Most of the HVX specific intrinsics/externs are just strings used in the backend.

Reply:
Perhaps not - they could just be const char *hvx_gather = "hvx_gather" somewhere. I didn't want to mess with things that much though - just trying to do a minimal drive-by cleanup in between meetings today